build/Build.Pack.cs

```diff
@@ -21,6 +21,8 @@
 
 partial class Build
 {
+    const string KubernetesTentacleContainerRuntimeDepsTag = "6.0-bookworm-slim";
```

Comment:
now we are building .NET 8, should this be `8.0-bookworm-slim`?

Reply:
Yes, good spot!